_includes/lottie.html

```diff
@@ -0,0 +1,5 @@
+<div
+    class="lottie"
+    data-lottie="{{include.lottie-path}}"
+    style="width:{{include.width}}px; height:{{include.height}}px;"
```

Comment:
This is not flexible enough. In the examples you have included for the formatting page, the lottie instances are fixed 400x400px. On a smaller phone, this breaks the layout as the animation is wider than the screen.

Similar to other images, we need to decide on the layout logic. Some ideas:
- Default to proportionally size it to the regular content width
- Optional max-width to make it smaller
- Ensure it can also be used in a desktop two-column layout (see "Image inline with the content")